securitycenter/snippets_v2/mute_findings_v2_test.py

```diff
@@ -67,14 +67,15 @@ def test_set_unmute_finding(finding):
     assert response.mute.name == "UNMUTED"
 
 
-# @backoff.on_exception(
-#     backoff.expo, (InternalServerError, ServiceUnavailable, NotFound), max_tries=3
-# )
-# def test_bulk_mute_findings(finding):
-#     # Mute findings that belong to this project.
-#     response = mute_findings_v2.bulk_mute_findings(
-#         f"organizations/{ORGANIZATION_ID}",
-#         "global",
-#         f'resource.project_display_name="{ORGANIZATION_ID}"',
-#     )
-#     assert response.done
+@backoff.on_exception(
+    backoff.expo, (InternalServerError, ServiceUnavailable, NotFound), max_tries=3
+)
+def test_bulk_mute_findings(finding):
+    # Mute findings that belong to this project.
+    filter_value = f'resource.project_display_name="{ORGANIZATION_ID}"'
+    response = mute_findings_v2.bulk_mute_findings(
+        f"organizations/{ORGANIZATION_ID}",
+        "global",
+        filter_value,
```

Comment:
For tests, use values directly rather than masking them under a variable.
```suggestion
        f"resource.project_display_name='{ORGANIZATION_ID}'",
```

Reply:
As per the chat, addressed this.